test/yulPhaser/TestHelpers.cpp

```diff
@@ -72,7 +72,7 @@ size_t phaser::test::countDifferences(Chromosome const& _chromosome1, Chromosome
 {
 	size_t count = 0;
 	for (size_t i = 0; i < min(_chromosome1.length(), _chromosome2.length()); ++i)
-		count += static_cast<int>(_chromosome1.optimisationSteps()[i] != _chromosome2.optimisationSteps()[i]);
+		count += static_cast<size_t>(_chromosome1.optimisationSteps()[i] != _chromosome2.optimisationSteps()[i]);
 
 	return count + abs(static_cast<int>(_chromosome1.length() - _chromosome2.length()));
```

Comment:
Unsigned underflow?

Reply:
Done.

Reply:
In the old code - yeah, but it has been changed in this commit.

But anyway, behavior in this case is well defined. It casts to a signed int properly.